services/pull/lfs.go

```diff
@@ -40,7 +40,7 @@ func LFSPush(ctx context.Context, tmpBasePath, mergeHeadSHA, mergeBaseSHA string
 	// 6. Take the output of cat-file --batch and check if each file in turn
 	// to see if they're pointers to files in the LFS store associated with
 	// the head repo and add them to the base repo if so
-	go createLFSMetaObjectsFromCatFileBatch(catFileBatchReader, &wg, pr)
+	go createLFSMetaObjectsFromCatFileBatch(db.DefaultContext, catFileBatchReader, &wg, pr)
```

Comment:
Why `default` if we have a `ctx` param?
```suggestion
	go createLFSMetaObjectsFromCatFileBatch(ctx, catFileBatchReader, &wg, pr)
```

Reply:
This is a goroutine, which could cause problems